src/commands/environments/list.ts

```diff
@@ -79,7 +79,7 @@ Use this command to list all environments available to a specific organization.`
       return
```

Comment:
In mix-command.ts, there is code in `outputHumanReadable()` that checks if the user provided an offset that is larger than the totalSize. When that happens, mix-cli is supposed to given an informative message about the upper bound that the 'offset' flag can take. However, in the command's own `outputHumanReadable()`, this is short-circuited by the the code finding that no results were returned, outputing a "No <object> found." message and returning.

Let's move this check for empty list inside the `outputHumanReadable` method of mix-command.ts. In order to remain able to give a granular message such as "No environments found.", let's add in the `transformResponse` method of the actual command a line that goes:

```
this.context.set('topic', 'environments')
```

so that `outputHumanReadable` in the base class can retrieve it and use it to build its `No ${topic} found.` message.

This will need to be done to other commands that use the 'offset' flag.